software/saltcorn.yml

```
@@ -0,0 +1,11 @@
+name: "Saltcorn"
+website_url: "https://saltcorn.com/"
+source_code_url: "https://github.com/saltcorn/saltcorn"
+description: "No-code database application builder for web and mobile applications. One platform for user interface, data backend, durable workflows, email, PDF generation, and AI applications"
```

```suggestion
description: "No-code database application builder for web and mobile applications. One platform for user interface, data backend, durable workflows, email, PDF generation, and AI applications."
```
Added a dot at the end